package.json

```diff
@@ -4,9 +4,18 @@
   "private": true,
   "dependencies": {
     "axios": "^0.18.0",
-    "react": "^16.4.1",
+    "bootstrap-table": "^1.12.1",
```

Comment:
Remove the unwanted libraries/components, make sure the packages are clear by keeping only the required and useful ones.